doc/m4/FS22-Group-36-M4-Report.md

```diff
@@ -0,0 +1,97 @@
+who played the highest cards draw another card and continue the next trick with these. This may continue until 
+there is only one highest rank in the trick.
+
+When all the tricks in the match are played out, the players count the number of their tricks.
+
+
+Points counting rules:\
+If a player announced the number of tricks correctly, then he gets the number of announced tricks squared as 
+positive points. Otherwise, the player gets the difference as negative points.
+
+Then the cards are distributed for the next match. The dealer duty moves one player to the right, the number 
+of cards that is distributed follows the sequence [5,4,3,2,1,2,3,4,5].
+
+For the match with only 1 card there is a special rule. The players don’t see their own card but put their card 
+with the front revealed on their forehead, so only the other players can see the card, but not the player himself. 
+After the 9 matches, the player with the highest score wins.
+
+## 2. Game view
+The order of the screenshots corresponds to the flow of the game, if possible. Matches 4, 3, 2 and the last match 
+are not listed. The only difference to the five-card match is the number of cards.
+Depends on the result of the game, after the last match, only the winner can see a screw your neighbor cup, 
+well-designed by Carmen. 
+
+
+## 3. Lessen learned
+We have divided our learnings into two chapters, Challenges and Notes on Teamwork.
+
+### 3.1 Challenges
+- The most challenging part was learning in an appropriate time the technology stack. 
+- The challenge was to review the pull requests in a reasonable amount of time and then to implement the reviews.
+While this was a time-consuming process, it helps to understand the code and how the program works.
+Thus, with each pull request, several people knew how far the user stories were implemented.
+- Another challenge was the feedback on the milestones. Some reviews were difficult to understand. 
+
+
+### 3.2 About teamwork
+- What we enjoyed the most was playing the game in real life on a map and then later online every week, which has two 
+huge advantages:
+    1. It supported our team building process
+    2. We test our game online every week
+
+- Weekly meetings of the whole group at the Irchel have simplified communication. Meeting in person has the advantage 
+that problems can be discussed bilaterally and the others can continue to work, but at the same time can listen with 
+one ear.
+	
+- The use of Git and the clarification of how we work with this tool has made it much easier to edit the software 
+efficiently, the main feature of Git we used was: 
+    1. Pull-request
+    2. Feature-Branch 
+
+- As a small team building event, where of course the subject SoPra was extensively discussed, we met for the
+"Sechseläuten". Coincidentally, there was a team member from the canton of Uri, which was also the guest canton.
```

Comment:
And the team member is also a firefighter. He was really impressed and fascinated by the spectacle and the later Cervelat grill on the fire.

Reply:
```suggestion
"Sechseläuten". Coincidentally, there was a team member from the canton of Uri, which was also the guest canton.
![image](https://user-images.githubusercontent.com/45465496/169947777-4eb655b9-e3f0-4c64-8a7a-de4156249687.png)
```